akka-docs/src/main/paradox/stream/stream-refs.md

```diff
@@ -1,4 +1,4 @@
-# StreamRefs - streams over network boundaries 
+# StreamRefs - Reactive Streams over the network
```

Comment:
I'm not sure this is a good change. You still have to use Akka Streams on both sides of the network.

Reply:
The main reason for changing it is for SEO/marketing. If someone has a question "how can I do reactive streams over the network", we want the number one Google result to be this page. When users land on the page, we want them to know that this is the page for them. This title does that. And Akka Streams is a reactive streams implementation. Of course you'll need something on each side, to do the actual communication, if it's not Akka Streams, it'll be whatever Reactor or RxJava provides to implement this if/when they do.

At the moment we have Reactor/RxJava people saying publicly that we're not really Reactive Streams because we only do reactive streams on the edges. In fact I got an email today that said exactly that from a core Reactor committer. They're not holding back at bending what it means to be reactive streams to suit their purposes to our detriment, so we shouldn't either, we need to be more aggressive in our messaging, otherwise they will walk all over us. There is nothing incorrect or misleading about saying that StreamRefs is Reactive Streams over the network - if you have a `Publisher` on one side, and a `Subscriber` on the other, following the instructions on this page will allow you to connect them, so let's sing that from the rooftops because it's an awesome feature that we have that other impls don't.

Reply:
ok, convinced

Reply:
I very much agree with James, they are bending the truth quite a lot, and being very aggressive about it. Which is rather disapointing... Us being more SEO friendly is totally fine, esp. given such aggresive behaviour from others.